test/Psy/Test/ConfigurationTest.php

```diff
@@ -238,4 +239,22 @@ public function testSetColorModeInvalid()
         );
         $config->setColorMode($colorMode);
     }
+
+    public function testSetCheckerInvalid()
+    {
+        $config = new Configuration();
+
+        $this->setExpectedException('TypeError');
```

Comment:
Oh, this is checking for a TypeError. This isn't going to work on anything less than PHP 7.